setup.py

```diff
@@ -62,6 +62,8 @@
 DOCLINES = __doc__.split('\n')
 
 REQUIRED_PKGS = [
+    # For iterative parsing JSON files
+    "ijson",
```

Comment:
We are very careful about adding new dependencies. I think this one could be in the `tests` or `dev` deps only.

Reply:
@thomwolf, done! I totally agree that one have to be cautious about adding new dependencies. The ideal option would have been that the Python Standard Library `json` package has an `iterload()` method (just as `xml.etree.ElemntTree` has `iterparse()`). But this is not the case, and thus the need to add an external dependency.

However, I would like to point out that when I decided to add it as a core dependency, I was thinking that maybe it should (or may) be used by dataset scripts (in `_generate_examples()` method) when dealing with large JSON files... Just keep this in mind! :)